code/modules/power/light_bulbs.dm

```diff
@@ -35,6 +35,9 @@
 	worn_icon_state = "c_tube"
 	brightness = 8
 
+/obj/item/light_bulb/tube/blue
```

Comment:
pretty sure this is the old naming scheme, and its not correctly setting base icon state, unless its intended to just be a normal tube?